package.json

```diff
@@ -15,14 +15,14 @@
     "astronomy"
   ],
   "dependencies": {
-    "nan": "^2.0.9",
-    "node-gyp": ">=1",
-    "merge": ">=1"
+    "nan": "^2.1.x",
+    "node-gyp": "3.0.x",
+    "merge": "^1.2.x"
   },
   "devDependencies": {},
   "optionalDependencies": {},
   "engines": {
-    "node": ">=0.12"
+    "node": "4.2.x"
```

Comment:
Changed to node >=0.12 (as it was before).

On Thu, Nov 5, 2015 at 11:16 AM, Samuel Gordalina <notifications@github.com>
wrote:

> In package.json
> <https://github.com/mivion/swisseph/pull/22#discussion_r44055184>:
>
> >    },
> >    "devDependencies": {},
> >    "optionalDependencies": {},
> >    "engines": {
> > -    "node": ">=0.12"
> > +    "node": "4.2.x"
>
> I dont agree that you force the engine to be 4.2, while it is acceptable
> to use 0.12
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mivion/swisseph/pull/22/files#r44055184>.
>